asmcli/commands/validate.sh

```diff
@@ -119,6 +119,14 @@ validate_hub() {
   if ! is_managed && [[ "${USE_VM}" -eq 1 && "${USE_HUB_WIP}" -eq 0 ]]; then
     fatal "Hub Workload Identity Pool is required to add VM workloads. Run the script with the -o hub-meshca option."
   fi
+
+  if ! is_managed && [[ "${CA}" == "mesh_ca" && "${USE_HUB_WIP}" -eq 0 ]] && ( can_register_cluster && is_gcp || is_cluster_registered ) ; then
+    info "Fleet workload identity pool is used as default for Mesh CA. "
+    context_set-option "USE_HUB_WIP" 1
```

Comment:
If the user does not specify `hub-meshca` option, `USE_HUB_WIP` will be set to 1 (use Hub WIP as default) when we know either the cluster has been registered into Hub or we can help user register the cluster. Otherwise, `USE_HUB_WIP` is still 0 and user can continue to use GKE WIP.

Reply:
If you're on GCP and you have workload identity enabled and you register the cluster, what's the difference between GKE and Fleet WIP?

Reply:
Hi Clay, in this case, there is no difference in WIP.